requirements/plugins.txt

```diff
@@ -1,14 +1,14 @@
-# change version ranges when upgrading from sumac
-tutor-android>=19.0.0,<20.0.0
-tutor-cairn>=19.0.0,<20.0.0
-tutor-credentials>=19.0.0,<20.0.0
-tutor-discovery>=19.0.0,<20.0.0
-tutor-ecommerce>=19.0.0,<20.0.0
-tutor-forum>=19.0.0,<20.0.0
-tutor-indigo>=19.0.0,<20.0.0
-tutor-jupyter>=19.0.0,<20.0.0
-tutor-mfe>=19.0.0,<20.0.0
-tutor-minio>=19.0.0,<20.0.0
-tutor-notes>=19.0.0,<20.0.0
-tutor-webui>=19.0.0,<20.0.0
-tutor-xqueue>=19.0.0,<20.0.0
\ No newline at end of file
+# For Tutor Main, we install plugins from their main branches instead of from PyPI
+tutor-android@git+https://github.com/overhangio/tutor-android@main
+tutor-cairn@git+https://github.com/overhangio/tutor-cairn@main
+tutor-credentials@git+https://github.com/overhangio/tutor-credentials@main
+tutor-discovery@git+https://github.com/overhangio/tutor-discovery@main
+tutor-ecommerce@git+https://github.com/overhangio/tutor-ecommerce@main
```

Comment:
Ecommerce is no longer maintained, please remove it from this list.

Reply:
done